test/deps/executables_test.dart

```diff
@@ -190,56 +190,4 @@ main() {
     test("are listed if --dev flag is set", _testAllDepsOutput('foo:bar'));
     test("are skipped if --no-dev flag is set", _testNonDevDepsOutput('\n'));
   });
-
-  group("lists dependencies overrides", () {
```

Comment:
I think it should still pass, the name of the dependencies doesn't change which is all we actually use, then we look up the package in the PackageGraph which should be overridden version.

If it doesn't pass, then I was wrong and we will need to do some more work to support this.